### PR TITLE
Added test for indexed payload attestation

### PR DESCRIPTION
**Pull request by Redidacove**
**What type of PR is this?**
Tests 

**What does this PR do? Why is it needed?**
Adds tests for indexed payload attestation

### CODE REVIEW

File: consensus-types/epbs/indexed_payload_attestation_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestGetAttestingIndices(t *testing.T) {
+	AttestingIndices := []primitives.ValidatorIndex{1, 2, 3}
```
Comment (terencechain) [inline consensus-types/epbs/indexed_payload_attestation_test.go:12]: Same for rest of the variables defined in this file

``` suggestion
	attestingIndices := []primitives.ValidatorIndex{1, 2, 3}
```

File: consensus-types/epbs/indexed_payload_attestation_test.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	NilAttestingIndices := []primitives.ValidatorIndex(nil)
```
Comment (terencechain) [inline consensus-types/epbs/indexed_payload_attestation_test.go:13]: Why nil?

**Review by terencechain — CHANGES_REQUESTED** [review]
Indexed payload attestation has nothing to do `[prysmaticlabs:payload_attestation_sync](https://github.com/prysmaticlabs/prysm/tree/payload_attestation_sync)`, I think it's fine  for epbs but the target branch should be right

File: consensus-types/epbs/indexed_payload_attestation_test.go (hunk 2 above)
Comment (Redidacove) [reply consensus-types/epbs/indexed_payload_attestation_test.go:13]: Should I make it empty slice

**Pull request closed without merging**